src/Security/Authentication/LoginManager.php

```diff
@@ -140,15 +134,6 @@ private function getRememberMeService(FirewallConfig $config)
             'security.authentication.rememberme.services.simplehash.' . $config->getName()
         ];
 
-        foreach ($definitions as $definition) {
-            if ($this->container->has($definition)) {
-                /** @var RememberMeServicesInterface $rememberMeService */
-                $rememberMeService = $this->container->get($definition);
-
-                return $rememberMeService;
-            }
-        }
-
-        return null;
+        return RememberMeServiceResolver::resolveRememberMeService($definitions);
```

Comment:
This service is likely not needed or in use anymore (delegated to new authenticator passport/[badge](https://github.com/symfony/symfony/blob/5.2/src/Symfony/Component/Security/Http/Authenticator/Passport/Badge/RememberMeBadge.php)).

I am on `symfony-http` 6.1.7 and this interface doesn't even exist anymore.
https://github.com/symfony/security-http/blob/142d48153a453dbd49e880eef6bc77e4ba162dff/RememberMe/RememberMeServicesInterface.php#L18

So this line, the resolver and the following code snippet are not even getting called or used at all and should be removed, if the PR is for a release that supports 11.x only or changed to be compatible with both guard authentication and new authenticator if it will support 10.5 too.

https://github.com/pimcore/customer-data-framework/blob/32d179987efcd4b7baad47af1692faaf4dc82090/src/Security/Authentication/LoginManager.php#L107-L109